db/migrate/20211101101117_create_posts.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePosts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :description
```

Comment:
šeit ar varēji droši atstāt to pašu string.

Bet citās db šim ir nozīme, tāpēc gluži nepareizi nav.